src/coreclr/debug/inc/amd64/primitives.h

```diff
@@ -70,7 +70,9 @@ constexpr CorDebugRegister g_JITToCorDbgReg[] =
     REGISTER_AMD64_R12,
     REGISTER_AMD64_R13,
     REGISTER_AMD64_R14,
-    REGISTER_AMD64_R15
+    REGISTER_AMD64_R15,
+    REGISTER_AMD64_RSP,
+    REGISTER_AMD64_RSP,
```

Comment:
is this related to this PR?

Reply:
No, this is just a temporary hack to work around a debugging issue I encountered - I'll take it out.